### PR TITLE
Remove __func__ from dev_dbg() logs

### PR DESCRIPTION
**Pull request by plbossart**

### CODE REVIEW

**Review by ranj063 — APPROVED** [review]

**Comment by plbossart** [comment]
@jsarha I don't think there's any existing way to add the same options from dev_dbg to dev_info.
That's probably why you see the use of the __func__ only for dev_info(), dev_warn and dev_dbg().

On one hand, one could argue that all logs should be formatted the same.
On the other hand, one could also make the argument that the default should be without __func__, and that dev_dbg() can add it automagically to help precisely debug.

I don't have a strong opinion, just following recommendations from higher powers in Linux land.

**Review by ujfalusi — APPROVED** [review]

File: sound/soc/sof/intel/hda-dsp.c [hunk 1]
```diff
@@ -743,7 +743,7 @@ int hda_dsp_resume(struct snd_sof_dev *sdev)
 			if (hlink->ref_count) {
 				ret = snd_hdac_ext_bus_link_power_up(hlink);
 				if (ret < 0) {
-					dev_dbg(sdev->dev,
+					dev_err(sdev->dev,
 						"error %d in %s: failed to power up links",
```
Comment (bardliao) [inline sound/soc/sof/intel/hda-dsp.c:747]: Do we want to use this opportunity to remove the "error" prefix?

Comment (plbossart) [reply sound/soc/sof/intel/hda-dsp.c:747]: @bardliao it's not really a prefix as in other logs (error: ), in this case the 'error' string is followed by an error code, that I don't think we want to remove. I suggest no change if that's alright with you.

Comment (bardliao) [reply sound/soc/sof/intel/hda-dsp.c:747]: > @bardliao it's not really a prefix as in other logs (error: ), in this case the 'error' string is followed by an error code, that I don't think we want to remove. I suggest no change if that's alright with you.

Yes, I am fine with that. I didn't notice that is not a prefix.

**Pull request merged**